### DISCUSSION TITLE
Next release, or workaround by temporarily changing event start date?

### DISCUSSION THREAD

**Discussion by paulmelis** [pretalx/pretalx#1788] [Q&A] [answered]
We're experiencing https://github.com/pretalx/pretalx/issues/1702 (session incorrectly scheduled at 12AM, without the option to delete it), which is really annoying as it's the first thing listed in the public schedule on mobile and can't be clicked for more info, so it probably confuses our audience at the event in a few weeks.

Is a release containing that fix scheduled before July, 1st? If not, would the following workaround work?

1. Change the event start date to be one day earlier
2. Remove incorrect session in the schedule editor
3. Set back event start date to correct date

I can't oversee if these steps would work with this bug, or what the consequences are for the (public) event info. I'd only need a few minutes to do these steps, and doing it at a moment when not many are online viewing the event info would be preferred, of course. Our CfP is long closed, we only occassionally have submitters update info, and we publish new schedules once every few days. But any other (automatic) consequences by temporarily changing event start date?

**Comment by rixx** [comment]
> Is a release containing that fix scheduled before July, 1st?

We don't have a fixed release schedule, but I don't anticipate a release within the next two weeks, no.

> But any other (automatic) consequences by temporarily changing event start date?

Changing an event's start date will under some conditions change the dates of existing sessions. If you change only the start date and keep the end date the same, this should be safe, off the top of my head.

**Reply by paulmelis** [reply, marked as answer]
FYI, the workaround worked! With the event start set to one day earlier I could now select the incorrect session and delete it. After putting back the start date and releasing a new schedule I saw no differences in the other session data.
